tests/core/converters/test_select.cpp

```diff
@@ -0,0 +1,33 @@
+#include <string>
+#include "gtest/gtest.h"
+#include "torch/csrc/jit/ir/irparser.h"
+#include "tests/util/util.h"
+#include "core/compiler.h"
+
+TEST(Converters, ATenSelectIntTwiceConvertsCorrectly) {

```

Comment:
Can we add the single select test case as well?

Reply:
sure